pallets/storage-provider/src/lib.rs

```diff
@@ -1015,6 +1065,36 @@ pub mod pallet {
     }
 
     impl<T: Config> Pallet<T> {
+        /// Gets the current deadline of the storage provider.
+        ///
+        /// If there is no Storage Provider of given AccountId returns [`Option::None`].
+        /// May exceptionally return [`Option::None`] when
+        /// conversion between BlockNumbers fails, but technically should not ever happen.
+        pub fn current_deadline(
+            storage_provider: &T::AccountId,
+        ) -> core::option::Option<primitives_proofs::CurrentDeadline<BlockNumberFor<T>>> {
```

Comment:
Is the `core::option::Option` needed here? Why can we not use just `Option`?

Reply:
Thinking about is further, I'm not sure the fully qualified name is required here. `Option` is usually imported as part of the prelude (if you run `cargo expand` you'll see what the prelude is) and as such, I would hope the compiler to be smart enough to import `Option` from the right prelude.

Reply:
I don't remember why I added it, but it looks like you're right guys. Works without it, removing.